2024-11-26 14:55:59,182 DEBUG    root            Loaded Command Group: ['gcloud', 'compute']
2024-11-26 14:55:59,213 DEBUG    root            Loaded Command Group: ['gcloud', 'compute', 'instances']
2024-11-26 14:55:59,234 DEBUG    root            Loaded Command Group: ['gcloud', 'compute', 'instances', 'describe']
2024-11-26 14:55:59,237 DEBUG    root            Running [gcloud.compute.instances.describe] with arguments: [INSTANCE_NAME: "bash-scripting"]
2024-11-26 14:56:11,049 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): compute.googleapis.com:443
2024-11-26 14:56:11,229 DEBUG    urllib3.connectionpool https://compute.googleapis.com:443 "GET /compute/v1/projects/inspired-photon-441609-k7/zones/us-central1-a/instances/bash-scripting?alt=json HTTP/1.1" 403 None
2024-11-26 14:56:11,233 DEBUG    root            Chosen display Format:list[title="Could not fetch resource:",always-display-title]
2024-11-26 14:56:11,234 DEBUG    root            (gcloud.compute.instances.describe) Could not fetch resource:
 - Required 'compute.instances.get' permission for 'projects/inspired-photon-441609-k7/zones/us-central1-a/instances/bash-scripting'
Traceback (most recent call last):
  File "/snap/google-cloud-cli/289/lib/googlecloudsdk/calliope/cli.py", line 998, in Execute
    resources = calliope_command.Run(cli=self, args=args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/289/lib/googlecloudsdk/calliope/backend.py", line 843, in Run
    resources = command_instance.Run(args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/289/lib/surface/compute/instances/describe.py", line 75, in Run
    return self._GetInstance(holder, instance_ref)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/289/lib/surface/compute/instances/describe.py", line 69, in _GetInstance
    return holder.client.MakeRequests([
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/289/lib/googlecloudsdk/api_lib/compute/client_adapter.py", line 145, in MakeRequests
    utils.RaiseToolException(
  File "/snap/google-cloud-cli/289/lib/googlecloudsdk/api_lib/compute/utils.py", line 139, in RaiseToolException
    RaiseException(problems, calliope_exceptions.ToolException, error_message)
  File "/snap/google-cloud-cli/289/lib/googlecloudsdk/api_lib/compute/utils.py", line 148, in RaiseException
    raise exception(
googlecloudsdk.calliope.exceptions.ToolException: Could not fetch resource:
 - Required 'compute.instances.get' permission for 'projects/inspired-photon-441609-k7/zones/us-central1-a/instances/bash-scripting'

2024-11-26 14:56:11,259 ERROR    root            (gcloud.compute.instances.describe) Could not fetch resource:
 - Required 'compute.instances.get' permission for 'projects/inspired-photon-441609-k7/zones/us-central1-a/instances/bash-scripting'

